### PR TITLE
Default sizes for tearout windows

### PR DESCRIPTION
**Pull request by owennw**

### CODE REVIEW

File: src/sidebars/favourites/tearout.js [hunk 1]
```diff
@@ -21,8 +21,8 @@
                             };
 
                             if (tearout) {
-                                config.minWidth = tileWidth;
-                                config.minHeight = tileHeight;
+                                config.maxWidth = tileWidth;
```
Comment (bjedrzejewski) [inline src/sidebars/favourites/tearout.js]: Why did you change maxWindths?

Comment (owennw) [inline src/sidebars/favourites/tearout.js]: That's the size of the card being torn out, and using the values as a max made more sense and fixed an odd bug too. Perhaps it should be both min and max though.

**Comment by akhmorgan** [comment]
looks good to me
